versioned_docs/version-3.0.0-beta/env-var.md

```diff
@@ -0,0 +1,54 @@
+---
+id: env-var
+title: Environment variable
+sidebar_label: Environment variable
+slug: /env-var
+---
+
+## Domain got Environment
```

Comment:
I think you should mention this on the domain page, not here. There is no context with the environment variable in this section.

Reply:
Also this is only applicable for centralised domains and subdomains